app/components/views/PrivacyPage/Privacy/Privacy.jsx

```diff
@@ -14,10 +14,27 @@ const Privacy = ({ isCreateAccountDisabled, setInterval }) => {
     accounts,
     accountMixerError,
     onStartMixerAttempt,
-    onGetPrivacyLogs
+    onGetPrivacyLogs,
+    allowSendFromUnmixed,
+    toggleAllowSendFromUnmixed
   } = usePrivacy();
 
   const [logs, setLogs] = useState("");
+  const [showingSendUnmixModal, showModal] = useState(false);
+  const onToggleSendFromUnmixed = () => {
```

Comment:
useCallback here might be useful

Reply:
it will be a callback with multiple function methods on its array dependency, which will be re-rendered each update. That's why I didn't add

Reply:
hrrmmm i'm not sure if i understand, useCallback will prevent extra re-renders in children components (the one receives `onToggleSendFromUnmixed`) it will guarantee to keep the same pointer to functions unless one of the deps array updates, in that case it will re-evaluate the useCallback value which will result in new pointer and children components wil re-render, without useCallback each re-render in parent will cause re-evaluation of the function which means new pointer which means children will re-render as well.

Reply:
Functions are comparing by reference, so if I add a function at the array dependency, it will not have much effect. Here: https://hackernoon.com/grasp-by-value-and-by-reference-in-javascript-7ed75efa1293

So I see no reason for adding it. We will not increase performance and the method will be more verbose.

Reply:
oh i see, didn't notice these are functions.